soh/soh/Enhancements/randomizer/3drando/hint_list/hint_list_exclude_dungeon.cpp

```diff
@@ -1960,9 +1959,9 @@ hintTextTable[RHT_SHADOW_TEMPLE_MAP_CHEST] = HintText(CustomMessage("They say th
                                                                                  /*french*/ "Selon moi, #derrière un bloc argent# dans le Gymnase Gerudo gît #[[1]]#.", {QM_RED, QM_GREEN}));
                                                                               // /*spanish*/ Según dicen, una #hazaña de fuerza# premia a las bandidas con #[[1]]#.
 
-    hintTextTable[RHT_GERUDO_TRAINING_GROUNDS_HEART] = HintText(CustomMessage("They say that a watching a #trial with Dinalfos# is #[[1]]#.",
-                                                                   /*german*/ "",
-                                                                   /*french*/ "Selon moi, dans #une épreuve avec un Dinalfos# révèle #[[1]]#.", {QM_RED, QM_GREEN}));
+    hintTextTable[RHT_GERUDO_TRAINING_GROUNDS_HEART] = HintText(CustomMessage("They say that a watching a #trial with Dinolfos# is #[[1]]#.",
+                                                                   /*german*/ "Man erzählt sich, daß das Sehen einer #Prüfung mit Dinolfos# #[[1]]# enthülle.",
```

Comment:
The German Dinolfos should be Dinodorus :P

![Screenshot_20250321-104432_Vivaldi~2](https://github.com/user-attachments/assets/bb1e7ecf-53a3-4a81-aaa3-264f7a7f823d)